test/unit/gateways/element_test.rb

```diff
@@ -142,6 +142,11 @@ def test_handles_error_response
     assert_failure response
   end
 
+  def test_successful_purchase_with_card_present_code
+    assert response = @gateway.purchase(@amount, @credit_card, @options.merge(card_present_code: 'Present'))
+    assert_equal 'Present', response.params["terminal"]["cardpresentcode"]
```

Comment:
Be sure to use single quotation marks here to keep RuboCop happy. 😁 
```suggestion
    assert_equal 'Present', response.params['terminal']['cardpresentcode']
```

Reply:
Whoops. Fixed:
https://github.com/activemerchant/active_merchant/blob/1aeb26ca1b256c188f8d483ed7ec31ee6cf37c45/test/unit/gateways/element_test.rb#L147